drive/modules/listener/watch_schedular.bal

```diff
@@ -39,26 +39,39 @@ class Job {
     
     private int retryCount = 1;
     private int retryScheduleCount = 1;
+    private int retryMaxCount;
+    private int retryInterval;
+    private int initialWaitTime;
 
     isolated function init(ListenerConfiguration config, drive:Client driveClient, 
                             Listener httpListener, HttpService httpService) {
         self.config = config;
         self.driveClient = driveClient;
         self.httpListener = httpListener;
         self.httpService = httpService;
+        self.retryMaxCount = config?.retryConfiguration?.retryCount ?: 20;
+        self.retryInterval = config?.retryConfiguration?.retryInterval ?: 100;
+        self.initialWaitTime = config?.retryConfiguration?.initialWaitTime ?: 300;
     }
 
     public isolated function execute() {
         error? err = self.registerWatchChannel();
         if (err is error) {
-            log:printWarn(WARN_CHANNEL_REGISTRATION, 'error = err);
-            if (self.retryCount <= 10) {
+            if (self.retryCount == 1){
+                log:printInfo(self.initialWaitTime.toString());
```

Comment:
This log does not have a meaning. Combining 
`log:printInfo(self.initialWaitTime.toString());` and `log:printInfo(CHANNEL_REGISTRATION_IN_PROGRESS);` we can say, 

`log:printInfo(CHANNEL_REGISTRATION_IN_PROGRESS. + "Waiting for " + initialWaitTime + "seconds to check result.")`

Reply:
Yeah. That's a good way. I will change accordingly.